kie-wb-parent/kie-wb-home-page-community/src/main/java/org/kie/workbench/client/home/AbstractHomeProducer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.workbench.client.home;
+
+import javax.inject.Inject;
+
+import com.google.gwt.core.client.GWT;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.kie.workbench.client.resources.i18n.Constants;
+import org.kie.workbench.common.screens.home.model.HomeModel;
+import org.kie.workbench.common.screens.home.model.HomeModelProvider;
+import org.kie.workbench.common.screens.home.model.ModelUtils;
+import org.uberfire.client.mvp.PlaceManager;
+
+import static org.kie.workbench.common.workbench.client.PerspectiveIds.*;
+import static org.uberfire.workbench.model.ActivityResourceType.PERSPECTIVE;
+
+public abstract class AbstractHomeProducer implements HomeModelProvider {
```

Comment:
```AbstractHomeProducer.java``` is showing up twice in this PR. They look identical though, so it seems only as a case of a duplicate.